knees4bees/mamminals
# Feature/images and icons

## What does this PR do?
 - Adds image files and icon links.
 
## How should it be tested?
 - Confirm that 8 cat images are present in the `assets` directory.
 - Open `index.html` in the browser and confirm that 8 icons are present.
